checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -26,9 +28,18 @@ def analyze_line(
         """
         This method first runs the keyword plugin. If it finds a match - it runs the entropy scanners, and if
         one of the entropy scanners find a match (on a line which was already matched by keyword plugin) - it is returned.
+        for source code files run and marge the tow plugin.
         """
+        entropy_matches = set()
         if len(line) <= MAX_LINE_LENGTH:
             keyword_matches = self.keyword_scanner.analyze_line(filename, line, line_number, **kwargs)
+            if f".{filename.split('.')[-1]}" in SOURCE_CODE_EXTENSION:
+                for entropy_scanner in self.high_entropy_scanners:
+                    matches = entropy_scanner.analyze_line(filename, line, line_number, **kwargs)
+                    if matches:
+                        entropy_matches = matches
```

Comment:
notice the variable entropy_matches is overridden each time that the the function arrive to line 40. is it expected?

Reply:
yes it's ok i need one of the high_entropy_scanners

Reply:
got it. I suggest to add 'break'. this way it is clearer that we need only one